chapters/grammar.adoc

```diff
@@ -171,6 +160,9 @@ _postfix_expression_.
 endif::ESSL[]
 ====
 
+_function_definition_ : ::
```

Comment:
This one I just moved up, it was weirdly hanging out at the very end of the grammar, so I put it with some other function definitions.